microsoft/botframework-solutions
# [TypeScript] Add script to update version and build all typescript projects

## Description
Add script to update version and build all typescript projects.
## Testing Steps
> NOTE: Run `npm install -g @microsoft/rush` to install the build tool globally.
1. Open powershell in the root directory of the project.
1. Run `./build.ts.ps1 -version "4.4.0"`
1. You will see the packages in `./outputpackages`

![image](https://user-images.githubusercontent.com/12394167/57140400-11a01c00-6d8e-11e9-81e0-5b8d992b101e.png)
